schema.sql

```diff
@@ -47,3 +47,22 @@ ADD CONSTRAINT animals_constraint_two
 FOREIGN KEY (owners_id)
 REFERENCES owners (id)
 ON DELETE CASCADE;
+
+CREATE TABLE vets(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(100),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE specializations(
+  name VARCHAR(100),
+  specialties VARCHAR(100)
```

Comment:
- [ ] Good work creating this table. However, I think it will be a good practice to use primary keys to make join tables. The reason being primary keys are unique. And in real cases, more than one person can have the same name.  Therefore, I will suggest you use primary keys here.

See example:
```
CREATE TABLE specializations (
   species_id int,  -- a reference to a species primary key
   vet_id int  -- a reference to a vet primary key
);
```

Reply:
- [ ] As suggested by @mmsesay, Kindly update the columns to use ids instead of names. This is because there may be more than one species or vet sharing the same name.  I have seen that you wrote a query to add an id column, instead, you should update the existing columns to use id. Kindly do the same for `visits`